### ISSUE TITLE
starting R

### ISSUE THREAD

**Issue by deb193** [ppanko/intro-to-hpc#2]
The Installing R packages locally document refers to "starting R" - but it would be good to put in the module load intel R&&R as well as explaining it.

**Comment by ppanko** [comment]
There's already an explanation on modules under [section 2 of the testJob-basic walkthrough](https://github.com/ppanko/intro-to-hpc/tree/master/testJob-basic#2-configuring-the-job) and the local R package installation guide already tells users how to load modules and start R [here](https://github.com/ppanko/intro-to-hpc/blob/master/local-R-packages.md#installing-packages-inside-r)

**Comment by deb193** [comment]
Yes. I see that now. I think some combination of trying to use the guide as a reference (under time pressure) and not a beginning to end tutorial - and the example you gave me with the && syntax. I had forgotten this was execute 2nd command if and only if the load command succeeded. What you sent me seemed alien and complicated syntax - some exoitc version of the load command. It took a few google searches to realize this was just two commands.

Also the TTU HPCC used to have a different way of preparing the environment and after logging on, one only typed R. This may have been a further challenge for me to "see" the new way that nieve guide users would not have had.

It is possible that a sentence discussing that R is started interactively the same as it is in batch jobs: 1) load modules, 2) R - that should provide extra support for the reader.
